Document/0x06b-Basic-Security-Testing.md

```diff
@@ -100,13 +102,12 @@ The iOS jailbreak scene evolves so rapidly that providing up-to-date instruction
 
 <img src="Images/Chapters/0x06b/cydia.png" alt="iOS App Folder Structure" width="250">
 
-Once you've jailbroken your iOS device and Cydia has been installed (as shown in the previous screenshot), proceed as follows:
+Once you've jailbroken your iOS device and either Cydia (see screenshot above) or Sileo has been installed, you can install the OpenSSH package. Once installed do the following:
```

Comment:
Maybe add that in the rest of the guide "Cydia" will be used, but Sileo should offer the same packages.

Reply:
done